### ISSUE TITLE
Datepicker fails when convertors in use

### ISSUE THREAD

**Issue by naxellar** [BorisMoore/jsviews#409]
I am using datepicker control from [jQuery UI tag controls library](https://www.jsviews.com/#download/jqueryui-tagcontrols).
```
{^{datepicker date _dateFormat="dd/mm/yy" convert="toDateString" convertBack="toWcfDate"}}
  <input type="text" value=""/>
{{/datepicker}}
```
`toDateString` converter returns date in datepicker format, `toWcfDate` converter gets datepicker format and returns ASP.NET format (e.g. `/Date(1198908717056-0700)/`).

[See JSFiddle demo](https://jsfiddle.net/naxellar/4hkjz2gy/)

This code fails on 34 line of `jsviews-jqueryui-widgets.js`:
```
Uncaught TypeError: Cannot read property 'getRsc' of undefined
    at getConverter (jsviews-jqueryui-widgets.js:35)
    at init (jsviews-jqueryui-widgets.js:367)
    at i.init (jsviews.js:183)
    at y (jsviews.js:776)
    at Function.eval [as fn] (eval at H (jsviews.js:2426), <anonymous>:6:2)
    at O (jsviews.js:1743)
    at Function.M [as render] (jsviews.js:1593)
    at ge (jsviews.js:4908)
    at Function.ue [as link] (jsviews.js:4862)
    at window.onload ((index):54)
```
```
/// jsviews-jqueryui-widgets.js
...
function getConverter(tag, cvt) {
  return cvt + "" === cvt ? tag.tagCtx.contentView.getRsc("converters", cvt) : cvt;
}
...
```

Did I do something wrong or it is a bug? Thank you.

**Comment by BorisMoore** [comment]
Thank you for calling this out! You are right, that looks like a bug in the jsviews-jqwueryui-widgets.js code.

I think the fix is to change that line to 

```
function getConverter(tag, cvt) {
  return cvt + "" === cvt ? tag.tagCtx.view.getRsc("converters", cvt) : cvt;
}
```
Can you test in your environment that that works for you?

**Comment by naxellar** [comment]
Yes, it works, thank you.

**Comment by BorisMoore** [comment]
This has been fixed in the v1.0.0 release
